src/goto-programs/goto_trace.h

```diff
@@ -201,7 +201,7 @@ class goto_tracet
     return steps.back();
   }
 
-  /// Returns the property IDs of all assertions in the trace
+  /// Returns the property IDs of all failed assertions in the trace
   std::vector<irep_idt> get_all_property_ids() const;
```

Comment:
Shouldn't the function then also be renamed? I concur that the passing properties wouldn't be of interest, but then the name of the function should also make this clear?